src/machinetalk/haltalk/haltalk_bridge.cc

```diff
@@ -142,11 +142,11 @@ static int bridge_fsm(htself_t *self, bridgeevent_t event)
     case BSTATE_RETRY_PROBE:
 
     case BSTATE_CONNECT:
-	bridge->z_bridge = zsocket_new (self->z_context, ZMQ_XSUB);
-	retval = zsocket_connect(bridge->z_bridge, self->cfg->bridgecomp_updateuri);
+	bridge->z_bridge = zsock_new (self->z_context, ZMQ_XSUB);
```

Comment:
These are just my notes - ignore

AFAICT, there is no `zsock_new()` method which takes 2 args ( or a context arg ).  Why does this compile?

Reply:
it does not, webtalk is disabled

Reply:
I realised that after I wrote.
It is a good example of why using 'clever' #if 0 constructs instead of just commenting a block out, is not a good idea.